data/model/new_releases_stat.py

```diff
@@ -0,0 +1,8 @@
+import pydantic
+from typing import List
+
+
+class NewReleasesStat(pydantic.BaseModel):
+    type: str
+    user_name: str
+    data: List
```

Comment:
Maybe add a docstring/docs for what will be in the list?

Reply:
Haven't documented field yet because we have not finalized on the contents of each item of the list.